LayoutTests/fast/dom/no-scroll-when-command-click-fragment-URL.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Fragment Navigation - ensure opening a fragment URL does not navigate on activating page.</title>
+<meta name="assert" content="This test checks command-clicking on a fragment URL does not navigate on the current page.">
+
+<head>
+<script src="../../resources/ui-helper.js"></script>
+
+<script>
+if (window.testRunner) {
+    testRunner.dumpAsText();
+    testRunner.waitUntilDone();
+}
+
+  
+ async function runTest()
+ {
+    if (!testRunner.runUIScript)
+        return;
+
+    var output = "";
+
+    var scrollOffset = window.pageYOffset;
+
+    UIHelper.activateElement(document.getElementById("LinkToClick"), ["metaKey"]); // metaKey = commandKey
+
+    await UIHelper.ensurePresentationUpdate();
+
+    if (window.pageYOffset != scrollOffset)
+      output += "FAIL: Page should not scroll. - " + window.pageYOffset;
+    else 
+      output += "PASS: Page did not scroll.";
+
+    var scrollOffset = window.pageYOffset;
+
+    var target = document.getElementById('target');
+    target.innerHTML = output;
+    target.style.paddingTop = "0px";
+
+    if (window.testRunner)
+        testRunner.notifyDone();
+  }
+
+  window.addEventListener('load', () => {
+    runTest();
+  }, false);
```

Comment:
Would be simpler as:

```suggestion
addEventListener('load', runTest);
```